topics/add-dependencies.md

```diff
@@ -3,18 +3,200 @@
+        implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:%coroutinesVersion%")
+    }
+    sourceSets["androidMain"].dependencies {
+    }
+    sourceSets["iosX64Main"].dependencies {
+        //SQLDelight will be available only in the iOS source set, but not in Android or common
+        implementation("com.squareup.sqldelight:native-driver:%sqlDelightVersion%)
+    }
+}
+```
+</tab>
+</tabs>
+
+### Dependency on another multiplatform project
+
+You can connect one multiplatform project to another as a dependency. To do this, simply add a project dependency to the source set that needs it. 
+If you want to use a dependency in all source sets, add it to the common one. In this case, other source sets will get their versions automatically.
+
+<tabs>
+<tab title="Groovy">
+    
+```Groovy
+kotlin {
+    sourceSets {
+        commonMain {
+            dependencies {
+                implementation project(':some-other-multiplatform-module')
+            }
+        }
+        androidMain {
+            dependencies {
+                //platform part of :some-other-multiplatform-module will be added automatically
+            }
+        }
+    }
+}
+```
+        
+</tab>
+<tab title="Kotlin">
+    
+```Kotlin
+kotlin {
+    sourceSets["commonMain"].dependencies {
+        implementation(project(":some-other-multiplatform-module"))
+    }
+    sourceSets["androidMain"].dependencies {
+        //platform part of :some-other-multiplatform-module will be added automatically
+    }
+}
+```
+</tab>
+</tabs>
 
 ## iOS dependencies
 
-Apple SDK dependencies (such as Foundation or Core Bluetooth) are available as a set of prebuilt libraries in a Kotlin Multiplatform Mobile project. They do not require any additional configuration.
+Apple SDK dependencies (such as Foundation or Core Bluetooth) are available as a set of prebuilt libraries in a Kotlin Multiplatform Mobile project. 
+They do not require any additional configuration.
 
-You can also reuse other libraries and frameworks from the iOS ecosystem in your iOS source sets. Kotlin supports interoperability with Objective-C dependencies and Swift dependencies if their API is exported to Objective-C with the `@objc` attribute. Pure Swift dependencies are not yet supported.
+You can also reuse other libraries and frameworks from the iOS ecosystem in your iOS source sets. 
+Kotlin supports interoperability with Objective-C dependencies and Swift dependencies if their API is exported to Objective-C with the `@objc` attribute. 
+Pure Swift dependencies are not yet supported.
+
+Integration with the CocoaPods dependency manager is also supported with the same limitation – you cannot use pure Swift pods. 
+
+We recommend:
+* [Using CocoaPods](#with-cocoapods) to handle iOS dependencies in Kotlin Multiplatform Mobile (KMM) projects. 
+* [Managing dependencies manually](#without-cocoapods) only if you want to tune the interop process specifically or if you have some other strong reason to do so.
```

Comment:
Are you sure about using the list here? Maybe it's just me but before reading the descriptions it looks like two **equally recommended** options, which is wrong.

Reply:
Yes, these are equally recommended options. And a bulleted list is used for a list of options. The list improves readability and adds the structure. People consume structured information easier than just text. And it's also a simple TOC of the section.
I don't get what's wrong with this list. Could you elaborate? :)

Reply:
Maybe this wording confused me, but I see:
1. `handle iOS dependencies in Kotlin Multiplatform Mobile (KMM) projects` - which is pretty common. For non-expert like me this description looks like a general way.
2. **only** if you want to tune the interop process specifically or if you have **some other strong reason to do so**. - which seems to me an advanced way for expert users; it can be potentially dangerous (**strong reason to do so** is scary)

Reply:
OK. I see your point now. removed the list – kept 2 sentences instead :)